schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -327,7 +327,7 @@ private SchemaIdVersion doAddSchemaVersion(String schemaName, SchemaVersion sche
         return new SchemaIdVersion(schemaMetadataInfo.getId(), readEntity(msg, Integer.class));
     }
 
-    private CatalogResponse readCatalogResponse(String msg) {
+    public static CatalogResponse readCatalogResponse(String msg) {
```

Comment:
readCatalogResponse is for parsing rest api calls. Users of schemaRegistryClient doesn't really need to understand the api calls it makes do we need to make this public.